src/containers/my-cooperations/cooperation-activities-list/CooperationActivitiesList.tsx

```diff
@@ -17,7 +17,7 @@ const CooperationActivitiesList = ({
   handleNonInputValueChange
 }: CooperationActivitiesListProps) => {
   const { selectedCourse, isAddedClicked, currentSectionIndex } =
-    useCooperationContext()
+    useAppSelector((state) => state.cooperations)
```

Comment:
Maybe we can create selector callback function in `cooperationsSlice.ts` and export it? This way we won't need to repeat the same `(store) => store.cooperations` in different places (for now repeated in `CourseSectionsList.tsx`, `CooperationActivitiesList.tsx` and `CooperationDetails.tsx`). What do you think?

Reply:
I agree with you. Created `cooperationsSelector` and used it like following:

```jsx
const {...} = useAppSelector(cooperationsSelector);
```